framework_tvb/tvb/core/adapters/abcadapter.py

```diff
@@ -562,7 +562,7 @@ def noise_configurable_parameters(self):
 
     def flaten_input_interface(self):
         # TODO: temporary condition to pass introspection on neoforms
-        form = self.get_form_class()()
+        form = self.get_form_class()
```

Comment:
Are you convinced this is correct?

Reply:
still used, but in deprecated methods related to the old input_tree